modules/installation-guide/partials/proc_using-openshift-web-console-to-install-che-on-codeready-containers.adoc

```diff
@@ -0,0 +1,49 @@
+[id="using-openshift-web-console-to-install-{prod-id-short}-on-codeready-containers_{context}"]
+= Using the OpenShift web console to install {prod-short} on CodeReady Containers
+
+.Prerequisites
+
+* An installation of CodeReady Containers. See link:https://cloud.redhat.com/openshift/install/crc/installer-provisioned[Installing CodeReady Containers].
+
+.Procedure 
+
+. To configure your host machine for CodeReady Containers, run:
++
+----
+$ crc setup
+----
+
+. To remove any previous cluster, run:
++
+----
+$ crc delete
+----
+
+. To start the CodeReady Containers virtual machine with at least 12 GB of RAM, run:
++
+----
+$ crc start --memory 12288
+----
+
+. When prompted, supply your user pull secret.
+
+. Take note of the password for the user `kubeadmin`, which the `crc start --memory 12288` command outputs.
+
+. To access the OpenShift web console, run:
++
+----
+$ crc console
+----
+
+. In the web console, log in with the username `developer` and the password `developer`. 
+
+. In the top right corner of the OpenShift web console, click *developer* -> *Log out*.
```

Comment:
You want to capitalize UI elements exactly as they're displayed in the UI. You very well may have done so here. But it strikes me as odd that _developer_ isn't capitalized but _Log out_ is. So simply double check that you have the capitalizations down. If so, no need to change :)

Reply:
`developer` starts with lower case, because it is the user name.

Reply:
Cool, leave as is then!